packages/neos-ui-views/src/NodeInfoView/style.css

```diff
@@ -4,21 +4,22 @@
     padding: 0;
     list-style: none;
 
-    li {
+    &__item {
         padding: 8px;
         border-bottom: 1px solid #222;
-        text-align: right;
         line-height: 20px;
         min-height: 20px;
 
         &:last-child {
             border-bottom: none;
         }
+    }
 
-        span {
-            float: left;
-            padding-right: 8px;
-            font-weight: bold;
-        }
+    &__title {
+        font-weight: bold;
+    }
+
+    &__content {
+        font-size: 13px;
```

Comment:
A bit smaller font size, not used currently in the UI, but I think it make sense here to avoid line break in UUID.

Reply:
Hmm I guess we can merge it as an exception, but in general I'd be cautious about ad-hoc adding of new font-sizes like this, without touching the stylguide and react-ui-components.